lib/rabbitmqctl.ex

```diff
@@ -154,7 +155,15 @@ defmodule RabbitMQCtl do
     opts
   end
 
-  defp validate_and_run_command(options, command, arguments) do
+  defp validate_and_run_command(command, arguments, options) do
+    validate_rabbit_app(command, options)
+      |> validate_command(command, arguments, options)
```

Comment:
Then, as suggested earlier, whether or not the app is running should not be treated as a validation and rather be considered an execution concern, like `badrpc`s. Otherwise you get two nearly identical error handling paths: genuine `badrpc`s and validations that "fail" because the app is stopped.